boost-maven/boost-maven-plugin/src/it/test-jaxrs-2.0/src/test/java/it/LibertyFeatureVersionIT.java

```diff
@@ -15,19 +15,82 @@
 import java.io.BufferedReader;
 import java.io.File;
 import java.io.FileReader;
+import java.util.Arrays;
 
 import org.junit.BeforeClass;
 import org.junit.Test;
+import org.eclipse.aether.repository.RemoteRepository;
+import org.eclipse.aether.RepositorySystem;
+import org.eclipse.aether.RepositorySystemSession;
+import org.eclipse.aether.artifact.Artifact;
+import org.eclipse.aether.resolution.VersionRangeRequest;
+import org.eclipse.aether.resolution.VersionRangeResult;
+import org.eclipse.aether.resolution.VersionRangeResolutionException;
+import org.eclipse.aether.DefaultRepositorySystemSession;
+import org.eclipse.aether.artifact.DefaultArtifact;
+import org.eclipse.aether.connector.basic.BasicRepositoryConnectorFactory;
+import org.eclipse.aether.impl.DefaultServiceLocator;
+import org.eclipse.aether.repository.LocalRepository;
+import org.eclipse.aether.spi.connector.RepositoryConnectorFactory;
+import org.eclipse.aether.spi.connector.transport.TransporterFactory;
+import org.eclipse.aether.transport.file.FileTransporterFactory;
+import org.eclipse.aether.transport.http.HttpTransporterFactory;
+import org.apache.maven.repository.internal.MavenRepositorySystemUtils;
 
 public class LibertyFeatureVersionIT {
 
     private static final String JAXRS_20_FEATURE = "<feature>jaxrs-2.0</feature>";
     private static String SERVER_XML = "target/liberty/wlp/usr/servers/defaultServer/server.xml";
+    private static String MESSAGES_LOG = "target/liberty/wlp/usr/servers/defaultServer/logs/messages.log";
+    private static String runtimeVersion;
 
     @BeforeClass
     public static void init() {
         String runtime = System.getProperty("boostRuntime");
         org.junit.Assume.assumeTrue("ol".equals(runtime) || "wlp".equals(runtime));
+        runtimeVersion = System.getProperty("libertyRuntimeVersion");
+    }
+
+    /*
+     * Go to Maven and figure out what the latest version of liberty runtime is out
+     * there. Use that version for comparing what level should be found in the
+     * messages.log file when the server is run. Only used when no libertyRunTime
+     * version is provided.
+     */
+    private String getLatesLibertyRuntimeVersion() {
```

Comment:
This is neat.. I didn't know how to do all this.   That said, I wonder if it creates some risk that we might not get the exact same latest version in certain environments (IDK, maybe when you have certain stuff in your local .m2 repo), and there isn't really a whole lot to the  boost layer that we're gaining coverage of here.   Boost would be doing the "right thing" even if we got back say 19.0.0.11  instead of 19.0.0.12.    I think we could lose this assertion.. but if we keep it, I would guard it with an error message that spells out in more detail that if it ever fails, well, it could be because of this range check, which doesn't mean Boost is broken, etc.